data/step_function_inputs/ESACCI_Biomass_L4_AGB_V4_100m_2017.json

```diff
@@ -0,0 +1,14 @@
+{
+  "collection": "ESACCI_Biomass_L4_AGB_V4_100m_2017",
+  "discovery": "inventory",
+  "inventory_url": "s3://maap-ops-workspace/jjfrench/CCI_2017.csv",
+  "upload": true,
+  "user_shared": false,
+  "cogify": true,
+  "csv_file_url_key": "s3_path",
+  "asset_roles": ["data"],
+  "asset_media_type": {
+          "tif": "image/tiff; application=geotiff; profile=cloud-optimized"
+    },
+  "asset_name": "tif"
```

Comment:
I think it's needed? We're stuck with one asset name otherwise. 

Just pushed a change that ensures cogify passes its inputs along to the next step.